### PR TITLE
Adding Profile Layout

### LINKED ISSUE #92: Create Profile Layout for app
Profile Layout needs to be added to show users details as in Event Brite app

Would You Like to Work on this issue?
Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #92
Profile Feature is added in app.Details of User would be shown in profile.
image

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@chashmeetsingh @imjog please Review!

**Comment by jogendra** [comment]
@SIMRAN1 I am getting this error, please look into it:
screen shot 2018-07-09 at 19 49 36

**Comment by jogendra** [comment]
Look like `Profile.storyboard` file is not referenced to project under **Copy bundle resources.**

**Review by jogendra — CHANGES_REQUESTED** [review]
After successfully login user should see the profile screen. Currently, login and signup screen is showing even after login. Please look into it.

**Comment by SIMRAN1** [comment]
@imjog it was working fine when I checked it. I have squashed my commits please see to it again!

**Comment by SIMRAN1** [comment]
Here is video demonstration https://drive.google.com/open?id=1wnMJI1TGUI4ZcjCBS4soHV6DGkaRKoL1

File: FOSSAsia/DecisionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class DecisionViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        if UserDefaults.exists(key: Constants.UserDefaultsKey.acessToken) {
```
Comment (jogendra) [inline FOSSAsia/DecisionViewController.swift]: Put code in a function and call that function here

**Review by jogendra — APPROVED** [review]
Refactor the code bit. otherwise looks fine to me

File: FOSSAsia/DecisionViewController.swift (hunk 1 above)
Comment (SIMRAN1) [reply FOSSAsia/DecisionViewController.swift]: okk @imjog !!

**Comment by jogendra** [comment]
@SIMRAN1 Ask review from Android and Web team as well!

**Comment by SIMRAN1** [comment]
@chashmeetsingh @nikit19 @simarsingh24 @sanyog please review!

**Review by nikit19 — APPROVED** [review]

**Pull request merged**